frontend/src/components/configurate-podcast/components/configurate-podcast-form/configurate-podcast-form.tsx

```diff
@@ -39,6 +39,8 @@ const ConfiguratePodcastForm: React.FC<Props> = ({
 
   const isFormDisabled = createPodcastStatus === DataStatus.PENDING;
 
+  const sortedGenres = genres.sort((a, b) => a.label > b.label ? 1 : -1);
```

Comment:
Does `label` field has string type? JavaScript `.sort` sorts strings by default and we can remove `(a, b) => a.label > b.label ? 1 : -1` .